webui/react/src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,193 @@
+import { Breadcrumb, Table } from 'antd';
+import React, { useCallback, useEffect, useState } from 'react';
+
+import ExperimentIcons from 'components/ExperimentIcons';
+import Grid, { GridMode } from 'components/Grid';
+import Link from 'components/Link';
+import Page from 'components/Page';
+import ProjectCard from 'components/ProjectCard';
+import Section from 'components/Section';
+import { experimentNameRenderer, relativeTimeRenderer } from 'components/Table/Table';
+import { paths } from 'routes/utils';
+import {
+  getCommands,
+  getExperiments,
+  getJupyterLabs,
+  getProjectsByUserActivity,
+  getShells,
+  getTensorBoards,
+} from 'services/api';
+import Icon from 'shared/components/Icon/Icon';
+import { dateTimeStringSorter } from 'shared/utils/sort';
+import { useAuth } from 'stores/auth';
+import { ShirtSize } from 'themes';
+import { CommandTask, DetailedUser, ExperimentItem, Project } from 'types';
+import { Loadable } from 'utils/loadable';
+
+const Dashboard: React.FC = () => {
+  const [experiments, setExperiments] = useState<ExperimentItem[]>([]);
+  const [tasks, setTasks] = useState<CommandTask[]>([]);
+  const [projects, setProjects] = useState<Project[]>([]);
+  const [submissions, setSubmissions] = useState<Submission[]>([]);
+  const [canceler] = useState(new AbortController());
```

Comment:
do you know why some places use useState but others use useRef in determined repo?
and how come useState is used here?  
i dont think it doesnt have to be useState

Reply:
my guess is that it's just out of syntactical preference (`.current` vs `[canceler]`, etc). to my knowledge there's no real downside of using `useState` here but deciding on a consistent way of doing this is a good idea.